src/compile.rs

```diff
@@ -144,9 +144,14 @@ pub fn compile(
 
     let mut shared_args = vec!["--manifest-path", context.manifest_path.to_str().unwrap()];
 
-    if python_feature != "" {
-        // This is a workaround for a bug in pyo3's build.rs
-        shared_args.extend(&["--features", &python_feature]);
+    match bindings_crate {
+        BridgeModel::Bindings(name) => {
+            if name == "pyo3" && python_feature != "" {
```

Comment:
Nitpick: This could be a single if with `binding_crate == BridgeModel::Bindings("pyo3".to_string())`

Reply:
I want to use fancy Rust pattern matching too badly :smile: I have updated the commit.